test/prototype_common_utils.py

```diff
@@ -628,21 +628,34 @@ def mark_framework_limitation(test_id, reason):
 
 
 class InfoBase:
-    def __init__(self, *, id, test_marks=None, closeness_kwargs=None):
+    def __init__(
+        self,
```

Comment:
This change just moves the comments to the signature as it is done in all other related classes.